docs/cvl/methods.md

```diff
@@ -706,12 +706,12 @@ The behavior of the `AUTO` summary depends on the type of call[^opcodes]:
   [State Mutability](https://docs.soliditylang.org/en/v0.8.12/contracts.html#state-mutability)
   in the Solidity manual for details.
 
-(function-summary)=
-#### Function summaries
+(expression-summary)=
+#### Expression summaries
```

Comment:
I'd leave the change minimal.
```suggestion
(function-summary)=
#### Function and Expression summaries
```

I think we want to leave the focus that `CVL functions` are used as summaries. From a spec writing perspective this is the cleaner solution.

Reply:
I'm actually fine with changing the refname as well.